### PR TITLE
move_nomic_atlas_for_user_query_visualization_into_separate_page

### PR DESCRIPTION
**Pull request by Maxwell-Lindsey**

### CODE REVIEW

**Comment by Maxwell-Lindsey** [comment]
PR is ready for review

**Review by KastanDay — APPROVED** [review]
Look very nice.

File: src/components/UIUC-Components/QueryAnalysisNavbar.tsx
Comment (KastanDay) [inline src/components/UIUC-Components/QueryAnalysisNavbar.tsx:1]: Another option is to have a single Navbar component, and make that component check which page we're on and display the proper buttons. But this way is simple and fine for now. Thanks!

**Review by KastanDay — APPROVED** [review]

**Comment by KastanDay** [comment]
@Maxwell-Lindsey I'm merging!

**Comment by Maxwell-Lindsey** [comment]
Awesome!

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Pull request merged**